### PR TITLE
Help page documentation and key admin functionality like user whitelisting

### PR DESCRIPTION
**Pull request by StreetsJ**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -43,63 +50,93 @@ const App = () => {
...
       .post("/api/v1/users", profile)
       .then((resp) => console.log(resp))
       .catch((resp) => console.log(resp));
+
+    const url = `/api/v1/users/${response.profileObj.googleId}`
+
+    axios
+      .get(url)
+      .then((resp) => {
+        console.log("Get admin data: ", resp.data.data.attributes.is_admin);
```
Comment (443SSL) [inline app/javascript/components/App.js:69]: great code formatting

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -43,63 +50,93 @@ const App = () => {
...
       {name == "" && (
-        <GoogleLogin
-          clientId="134632541809-skppjomtgttr7vkb08lmoki4p15nv9d5.apps.googleusercontent.com"
-          onSuccess={responseGoogle}
-          onFailure={responseGoogle}
-          cookiePolicy={"single_host_origin"}
-        />
+        <Fragment>
+          <h2>
+            Welcome to the APAMSA App! Please sign in
```
Comment (443SSL) [inline app/javascript/components/App.js:86]: good labeling this login

File: app/javascript/components/App.js [hunk 3]
```diff
@@ -43,63 +50,93 @@ const App = () => {
...
+                </Button>
+              }
+            />
+          </Card>
+          <Router>
+            <Switch>
+              <Route exact path="/" component={HomePage} />
+              <Route path="/blog">
+                <BlogPage />
+              </Route>
```
Comment (443SSL) [inline app/javascript/components/App.js:115]: good nav bar

File: app/javascript/components/ShowUsersPage/User.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+          <Fragment>
+            <CardContent>
+              <Typography>
+                Admin
+              </Typography>
+            </CardContent>
+            <CardActions>
+              <Button
+                variant="contained"
+                endIcon={<RemoveIcon />}
```
Comment (443SSL) [inline app/javascript/components/ShowUsersPage/User.js:76]: Excellent user model

File: app/javascript/components/Tabs/helpPage.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
+import { Button, CardContent, CardHeader, Paper, Typography } from "@mui/material";
+import React, { Fragment } from "react";
+import TabBar from "./tabBar";
+
+const HelpPage = () => {
+  const title1 = "Welcome to the APAMSA help page";
+  const subtitle1 = "Here you will find information about the pages and the user heirarchy.";
+
```
Comment (pierre-vu) [inline app/javascript/components/Tabs/helpPage.js:8]: Good work on help page

File: app/javascript/components/ShowUsersPage/User.js [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+      </Paper>
+      <Dialog
+        open={openUpdated}
+        onClose={() => {
+          setOpenUpdated(false);
+        }}
+      >
+        <DialogTitle id="alert-dialog-title">
+          <Alert>{name} privelege was successfully changed!</Alert>
+        </DialogTitle>
```
Comment (443SSL) [inline app/javascript/components/ShowUsersPage/User.js:116]: Good authentication controls for admin.

File: app/javascript/components/Tabs/tabBar.js [hunk 7]
```diff
@@ -14,6 +17,10 @@ const TabBar = (props) => {
         <Tab value={2} component={Link} to="/events" label="Events" />
         <Tab value={3} component={Link} to="/about" label="About Us" />
         <Tab value={4} component={Link} to="/profile" label="My Profile" />
+        <Tab value={5} component={Link} to="/help" label="Help" />
+        {is_admin && (
+          <Tab value={6} component={Link} to="/users" label="Users List" />
+        )}
```
Comment (pierre-vu) [inline app/javascript/components/Tabs/tabBar.js:23]: Help page on navbar

File: app/javascript/components/Tabs/helpPage.js [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  const adminProf = "My Profile: Admins can view their profile that displays their information.";
+  const adminUser = "User List: Admins can view a users list page that displays all users that login to the application. They can then choose whether they want to grant or revoke admin privileges to that user.";
+  const adminPrivs = [adminHome, adminBlog, adminEvent, adminAbout, adminProf, adminUser];
+
+  const user_list = userPrivs.map( item => {
+    return (<li key={item}>{item}</li>)
+  })
+
+  const admin_list = adminPrivs.map( item => {
+    return (<li key={item}>{item}</li>)
```
Comment (443SSL) [inline app/javascript/components/Tabs/helpPage.js:29]: Great help page!

File: app/javascript/components/Tabs/profilePage.js [hunk 9]
```diff
@@ -36,6 +38,13 @@ const ProfilePage = () => {
           </CardContent>
         </Card>
       )}
+      {name === "" && (
+        <Card>
+          <CardHeader
```
Comment (443SSL) [inline app/javascript/components/Tabs/profilePage.js:43]: good card

**Review by 443SSL — APPROVED** [review]

**Comment by 443SSL** [comment]
Great commit good adding the help page and the formatting looks great

**Pull request merged**